.github/workflows/smoke-test-daily.yml

```diff
@@ -1,99 +1,173 @@
 name: Smoke test daily
 on:
-    schedule:
-        - cron: '25 3 * * *'
+    # mytodo remove pull_request, uncomment cron
+    # schedule:
+    #     - cron: '25 3 * * *'
     workflow_dispatch:
```

Comment:
I disabled this cron trigger for the meantime until this PR already passes the review. I'll re-enable it before merging.